spec/models/item_spec.rb

```diff
@@ -12,17 +12,30 @@
     it { should validate_presence_of(:description) }
     it { should validate_presence_of(:unit_price) }
     it { should validate_numericality_of(:unit_price) }
-    it { should validate_presence_of(:enabled) }
   end
 
-  # before :each do
-  #
-  # end
-  #
-  # describe 'class methods' do
-  #   describe '.' do
-  #   end
-  # end
+  before :each do
+    @merchant1 = create(:merchant)
+    @merchant2 = create(:merchant)
+
+    @item1 = create(:item, enabled: true, merchant: @merchant1)
+    @item2 = create(:item, enabled: true, merchant: @merchant1)
+    @item3 = create(:item, enabled: false, merchant: @merchant1)
+  end
+
+  describe 'class methods' do
+    describe '.all_enabled' do
+      it 'can return ONLY items where enabled is true' do
+        expect(Item.all_enabled).to eq([@item1, @item2])
+      end
+    end
+
+    describe '.all_disabled' do
+      it 'can return ONLY items where enabled is false' do
+        expect(Item.all_disabled).to eq([@item3])
+      end
+    end
```

Comment:
These tests look good!  A couple of things you could consider:

1. Add an extra `expect` line for each test to confirm the method doesn't return the other item.  It is implied with how you are testing it, so it is good as is, but might just further demonstrate the method is working as expected by testing what the method **doesn't** return.
2. You could consider using third-person present tense for the `it` block names: https://rspec.rubystyle.guide/#should-in-example-docstrings

Reply:
Updated all the tenses of the it blocks and added test to confirm what it's not returning